l10n_it_fatturapa_out/migrations/16.0.1.0.9/pre-migrate.py

```diff
@@ -19,8 +19,9 @@ def migrate(env, version):
         """,
         (table_name, "%s" % constraint_name),
     )
+    result = env.cr.fetchall()
 
-    if env.cr.fetchall()[0]:
+    if result and result[0]:
         alter_table_sql = sql.SQL("ALTER TABLE {} DROP CONSTRAINT {}").format(
```

Comment:
Forse si può semplificare il tutto con
`ALTER TABLE wizard_export_fatturapa DROP CONSTRAINT IF EXIST wizard_export_fatturapa_report_print_menu_fkey`.
Così la query precedente manco serve, giusto?
(ho verificato, IF EXIST è stato introdotto in pg 9.0, Odoo 16 richiede minimo 10).